packages/croncat-sdk-agents/src/msg.rs

```diff
@@ -0,0 +1,60 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Timestamp, Uint128, Uint64};
+
+use crate::types::AgentStatus;
+
+#[cw_serde]
+pub struct AgentContractInstantiateMsg {}
+
+#[cw_serde]
+pub enum AgentExecuteMsg {
+    RegisterAgent {
+        payable_account_id: Option<String>,
+        cost: u128,
+    },
+    UpdateAgent {
+        payable_account_id: String,
+    },
+    CheckInAgent {},
+    UnregisterAgent {
+        from_behind: Option<bool>,
+    },
+    WithdrawReward {},
```

Comment:
```suggestion
    WithdrawReward {},
    NewTaskCreated {},
```
Can you implement please this logic: 
https://github.com/CronCats/cw-croncat/blob/b920d1a5d5b1730397b04acdbba205eacfd487bd/contracts/cw-croncat/src/tasks.rs#L380-L395

Reply:
√